### PR TITLE
Norm cn 20

### PR DESCRIPTION
**Pull request by frankwhzhang**
add 2.0 api cn doc for batch_norm instance_norm layer_norm and group _norm

### CODE REVIEW

**Comment by frankwhzhang** [comment]
屏幕快照 2020-08-27 下午4 00 32

屏幕快照 2020-08-27 下午4 00 44

屏幕快照 2020-08-27 下午4 00 57

**Comment by frankwhzhang** [comment]
屏幕快照 2020-08-27 下午4 01 32

屏幕快照 2020-08-27 下午4 01 55

屏幕快照 2020-08-27 下午4 02 25

File: doc/fluid/api_cn/nn_cn/BatchNorm3d_cn.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. _cn_api_nn_BatchNorm3d:
+
+BatchNorm3d
+-------------------------------
+
+.. py:class:: paddle.nn.BatchNorm3d(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCDHW', track_running_stats=True, name=None):
+
+
+该接口用于构建 ``BatchNorm1d`` 类的一个可调用对象，具体用法参照 ``代码示例`` 。可以处理4D的Tensor, 实现了批归一化层（Batch Normalization Layer）的功能，可用作卷积和全连接操作的批归一化函数，根据当前批次数据按通道计算的均值和方差进行归一化。更多详情请参考 : `Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift <https://arxiv.org/pdf/1502.03167.pdf>`_
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/BatchNorm3d_cn.rst]: BatchNorm3d

File: doc/fluid/api_cn/nn_cn/BatchNorm1d_cn.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    - **name** (string, 可选) – BatchNorm的名称, 默认值为None。更多信息请参见 :ref:`api_guide_Name` 。
+
+
+返回：无
+
+形状：
+    - input: 形状为（批大小，通道数）的2-D Tensor 或（批大小, 通道数，长度）的3-D Tensor。
+    - output: 和输入形状一样。
+
+说明：
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/BatchNorm1d_cn.rst]: 可以用Note的格式~

File: doc/fluid/api_cn/nn_cn/functional_cn/batch_norm_cn.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    running_variance = np.random.random(size=1).astype('float32')
+    weight_data = np.random.random(size=1).astype('float32')
+    bias_data = np.random.random(size=1).astype('float32')
+    x = paddle.to_tensor(x)
+    rm = paddle.to_tensor(running_mean)
+    rv = paddle.to_tensor(running_variance)
+    w = paddle.to_tensor(weight_data)
+    b = paddle.to_tensor(bias_data)
+    batch_norm_out = paddle.nn.functional.batch_norm(x, rm, rv, w, b)
+    print batch_norm_out
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/functional_cn/batch_norm_cn.rst]: 少了 （） 不一定能过CI 下同

File: doc/fluid/api_cn/nn_cn/InstanceNorm2d_cn.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    \mu_{\beta}        &\gets \frac{1}{m} \sum_{i=1}^{m} x_i                                 \quad &// mean  \\
+    \sigma_{\beta}^{2} &\gets \frac{1}{m} \sum_{i=1}^{m}(x_i - \mu_{\beta})^2               \quad &// variance \\
+    \hat{x_i}          &\gets \frac{x_i - \mu_\beta} {\sqrt{\sigma_{\beta}^{2} + \epsilon}}  \quad &// normalize \\
+    y_i &\gets \gamma \hat{x_i} + \beta                                                      \quad &// scale-and-shift
+
+Note:
+    `H` 是高度, `W` 是宽度.
+
+
+参数：
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/InstanceNorm2d_cn.rst:23]: 参数列表顺序需要和定义一致，即data_format 与 track_running_stats的位置关系

File: doc/fluid/api_cn/nn_cn/InstanceNorm1d_cn.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    \sigma_{\beta}^{2} &\gets \frac{1}{m} \sum_{i=1}^{m}(x_i - \mu_{\beta})^2               \quad &// variance \\
+    \hat{x_i}          &\gets \frac{x_i - \mu_\beta} {\sqrt{\sigma_{\beta}^{2} + \epsilon}}  \quad &// normalize \\
+    y_i &\gets \gamma \hat{x_i} + \beta                                                      \quad &// scale-and-shift
+
+
+Note:
+    `H` 是高度, `W` 是宽度.
+
+
+参数：
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/InstanceNorm1d_cn.rst:24]: 参数位置关系，参数列表的顺序应和定义一致，data_format、track_running_stats的先后关系；

File: doc/fluid/api_cn/nn_cn/InstanceNorm3d_cn.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    \mu_{\beta}        &\gets \frac{1}{m} \sum_{i=1}^{m} x_i                                 \quad &// mean  \\
+    \sigma_{\beta}^{2} &\gets \frac{1}{m} \sum_{i=1}^{m}(x_i - \mu_{\beta})^2               \quad &// variance \\
+    \hat{x_i}          &\gets \frac{x_i - \mu_\beta} {\sqrt{\sigma_{\beta}^{2} + \epsilon}}  \quad &// normalize \\
+    y_i &\gets \gamma \hat{x_i} + \beta                                                      \quad &// scale-and-shift
+
+Note:
+    `H` 是高度, `W` 是宽度.
+
+
+参数：
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/InstanceNorm3d_cn.rst:22]: 同上

File: doc/fluid/api_cn/nn_cn/BatchNorm3d_cn.rst (hunk 1 above)
Comment (frankwhzhang) [reply doc/fluid/api_cn/nn_cn/BatchNorm3d_cn.rst]: fix

File: doc/fluid/api_cn/nn_cn/BatchNorm1d_cn.rst (hunk 2 above)
Comment (frankwhzhang) [reply doc/fluid/api_cn/nn_cn/BatchNorm1d_cn.rst]: fix

File: doc/fluid/api_cn/nn_cn/functional_cn/batch_norm_cn.rst (hunk 3 above)
Comment (frankwhzhang) [reply doc/fluid/api_cn/nn_cn/functional_cn/batch_norm_cn.rst]: fix

File: doc/fluid/api_cn/nn_cn/InstanceNorm2d_cn.rst (hunk 4 above)
Comment (frankwhzhang) [reply doc/fluid/api_cn/nn_cn/InstanceNorm2d_cn.rst:23]: fix

File: doc/fluid/api_cn/nn_cn/InstanceNorm1d_cn.rst (hunk 5 above)
Comment (frankwhzhang) [reply doc/fluid/api_cn/nn_cn/InstanceNorm1d_cn.rst:24]: fix

File: doc/fluid/api_cn/nn_cn/InstanceNorm3d_cn.rst (hunk 6 above)
Comment (frankwhzhang) [reply doc/fluid/api_cn/nn_cn/InstanceNorm3d_cn.rst:22]: fix

**Review by TCChenlong — APPROVED** [review]
LGTM
